url-gen/src/main/kotlin/com/cloudinary/transformation/transcode/TranscodeActions.kt

```diff
@@ -283,6 +291,41 @@ class H264Codec private constructor(private val profile: VideoCodecProfile? = nu
     }
 }
 
+class H265Codec private constructor(private val profile: VideoCodecProfile? = null, private val level: Any? = null, private val bFrames: Boolean = true) :
+    VideoCodec("h265") {
+    override fun toString(): String {
+        var bFramesString: String? = null
+        if (!bFrames) {
+            bFramesString = "bframes_no"
+        }
+        return codec.joinWithValues(profile, level, bFramesString)
+    }
+
+    class Builder {
+        private var profile: VideoCodecProfile? = null
+        private var level: Any? = null
+        private var bFrames: Boolean = true
+
+        fun bFrames(bFrames: Boolean) = apply {
+            setAutoProfileAndLevel(bFrames)
+            this.bFrames = bFrames
+        }
+
+        private fun setAutoProfileAndLevel(setAutoProfleAndLevel: Boolean) {
+            if (setAutoProfleAndLevel) {
+                this.profile = null;
+                this.level = null
+            } else {
+                this.profile = VideoCodecProfile.auto()
+                this.level = "auto"
```

Comment:
Why not `VideoCodecLevel.auto()`?

Reply:
Unlike profile which is type `VideoCodecProfile` level is `Any` and that's why I gave it a `String` value

Reply:
I see no reason not to add it to the enum